test/remote/gateways/remote_trans_first_test.rb

```diff
@@ -70,12 +70,12 @@ def test_failed_purchase
     assert_equal 'Insufficient funds', response.message
   end
 
-  def test_successful_refund
```

Comment:
Well, this is where remote tests get tricky since we sort of have to work around how a gateway's sandbox is functioning. We should still test that it returns the right message, is successful, etc., but just know (or write a short comment to explain) that we need to wait for the transaction to settle which could take ~24 hours. It's unfortunate that the full remote test will never fully pass in one run but the test will still need to assert the response we want assuming the transaction was settled.